pkgs/development/tools/misc/swig/3.x.nix

```diff
@@ -17,8 +17,8 @@ stdenv.mkDerivation rec {
 
   configureFlags = [ "--without-tcl" ];
 
+  # Disable ccache documentation as it needs yodl
```

Comment:
Just fixing a small grammar mistake while I'm here, and moving the comment out of the `postPatch` so it can be updated without a rebuild.